ardere.json

```diff
@@ -0,0 +1,31 @@
+{
+  "ecs_name": "rpapa-demo",
```

Comment:
we'll probably want to create 1 ecs cluster for each project under test.  this will need to be done manually.